src/config.ts

```diff
@@ -1,14 +1,15 @@
-import { fileURLToPath } from 'node:url'
-import { join } from 'pathe'
-import { defu } from 'defu'
 import { defineConfig as defineVitestConfig } from 'vitest/config'
 import { type UserConfig as ViteUserConfig, mergeConfig } from 'vite'
-import { NITRO_OUTPUT_DIR } from './constants'
+import defu from 'defu'
+import { loadOptions as loadNitroOptions } from 'nitropack'
 
 export interface NitroInlineConfig {
```

Comment:
Can we bring back `mode?: 'development' | 'production'`? I want to let the users test the dev vs. prod Node bundle.